src/lib.rs

```diff
@@ -7,18 +7,22 @@
 //! `futures::Stream`.
 extern crate libc;
 extern crate mio;
+extern crate bytes;
+extern crate tokio_io;
 extern crate tokio_core;
 
 use std::io;
 use std::os::unix::io::{AsRawFd, RawFd};
+use bytes::{ BufMut, BytesMut };
```

Comment:
Rust coding convention does not allow spaces after `{` or before `}` in the imports.